src/instruments/src/ND/pages/ArcMode.tsx

```diff
@@ -465,7 +464,7 @@ const Overlay: React.FC<OverlayProps> = memo(({ heading, track, rangeSetting, si
 
             <TrackBug heading={heading} track={track} />
             { lsDisplayed && <IlsCourseBug heading={heading} ilsCourse={ilsCourse} /> }
-            <SelectedHeadingBug heading={heading} selected={selectedHeading} />
+            <SelectedHeadingBug heading={Math.round(heading)} selected={selectedHeading} />
```

Comment:
Don't do this. It causes the bug to jump rather than rotate smoothly with the heading.